### DISCUSSION TITLE
i don't have folder layout title sider in my project

### DISCUSSION THREAD

**Discussion by Codeine999** [refinedev/refine#3915] [Q&A] [answered]
i need to edit siderbar but i don't have that folder in my components folder have only folder header and file index.ts i don't know how to do

**Comment by aliemir** [comment, marked as answer]
Hey @Codeine999, by default, those components are used from the UI package you've selected. You can use `@refinedev/cli`'s `swizzle` command to export those components into your project and customize it all you want. Check out this doc for more info about the CLI and `swizzle` https://refine.dev/docs/packages/documentation/cli/#swizzle

**Reply by Codeine999** [reply]
thank so much<3 and we must edit path Layout  from @refinedev/cli to components/layout in app.tsx
